crates/sui-rpc-api/proto/sui/rpc/alpha/proof_service.proto

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+package sui.rpc.alpha;
+
+import "sui/rpc/v2/object_reference.proto";
+
+// ProofService provides cryptographic proofs for blockchain objects.
+service ProofService {
+  // Returns an inclusion proof if a specified object id was written in a specified checkpoint.
+  rpc GetObjectInclusionProof(GetObjectInclusionProofRequest) returns (GetObjectInclusionProofResponse);
+}
+
+// Object Checkpoint State inclusion proof.
+message OCSInclusionProof {
```

Comment:
I have a question re how one designs these APIs: is there a need to surface the internal details of a proof at the gRPC level? Why not just mark it as a sequence of bytes? I like that approach because it would in turn simplify the logic used to deserialize the protobuf structs. I don't know much about gRPC API design; so not sure what's the recommended approach.